include/rem_aufile.h

```diff
@@ -26,3 +26,5 @@ int aufile_read(struct aufile *af, uint8_t *p, size_t *sz);
 int aufile_write(struct aufile *af, const uint8_t *p, size_t sz);
 size_t aufile_get_size(struct aufile *af);
 size_t aufile_get_length(struct aufile *af, struct aufile_prm *prm);
+int aufile_set_position(struct aufile *af, struct aufile_prm *prm,
+		off_t pos_ms);
```

Comment:
it is probably better to use `size_t` instead of `off_t` here ...

Reply:
I did that first, but it's `fseek(FILE *, long, int);` and using `size_t` breaks the Windows build.